src/ONE.Models.CSharp/BuildSubmodule.bat

```diff
@@ -1,3 +1,8 @@
+# Clean up temporary flat proto files and previous generted cs files for a clean generation
+IF EXIST "..\ONE.Interfaces.ProtocolBuffers\proto\flat" rd /q /s "..\ONE.Interfaces.ProtocolBuffers\proto\flat"
+IF EXIST "generated" rd /q /s "generated"
```

Comment:
When split dataset_v2.proto file to multiple files, I got some weird compiling errors. Turns out garbage files are remained under the `flat` folder and `generated` folder, better to clean them up to make the generation script re-runable.

Reply:
Nice change here, I always just raged internally and manually deleted the contents of the folders